tests/src/JIT/Regression/JitBlue/DevDiv_200492/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-23816",
+    "System.Console": "4.0.0-rc2-23816",
+    "System.Runtime": "4.1.0-rc2-23816",
+    "System.Runtime.Extensions": "4.0.10",
+    "System.Runtime.InteropServices": "4.1.0-rc2-23816",
+    "System.Reflection.Extensions": "4.0.1-rc2-23816"
```

Comment:
This is only one that appears missing in ```extra\project.json```.
Can you add this and use the file in the project file instead of adding this one in this test?